mmedit/apis/colorization_inference.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmengine.dataset import Compose
+from mmengine.dataset.utils import default_collate as collate
+from torch.nn.parallel import scatter
+
+
+def colorization_inference(model, img):
+    """Inference image with the model.
```

Comment:
Type lint in the function signature is missing. (Maybe can convert to a **good first issue**)